utility/src/main/java/org/oppia/android/util/parser/ImageLoader.kt

```diff
@@ -36,12 +36,12 @@ interface ImageLoader {
   )
 
   /**
-   * Loads the specified [imageDrawable] resource into the specified [target].
+   * Loads the specified [imageDrawable] resource into the specified [imageView].
    * Optional [transformations] may be applied to the image.
    */
   fun loadDrawable(
     @DrawableRes imageDrawableResId: Int,
-    target: ImageTarget<Drawable>,
+    imageView: ImageView,
```

Comment:
Why the change away from ``ImageTarget``? That was more consistent with the other loader methods. (If it was for testing purposes you can retrieve the underlying image view from the target in the fake/test utility class).

Reply:
It was for testing purposes, changed it back now.